database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Models/PostListViewModel.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  Copyright (c) 2021 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
```

Comment:
The should not be dependencies on SwiftUI in a ViewModel. You may need to import `Combine` instead to make `@Published` property wrapper available.

Reply:
That make sense. I was wondering should I also make this change for other files such as `UserViewModel.swift`? If so, after changing from `import SwiftUI` to `import Combine`, `@AppStorage` becomes an unknown attribute, what should I import to avoid this error?

Reply:
Yes, it should be done for other view models. Let me take a looks if I can suggest anything about `@AppStorage`...